src/sentry/integrations/jira/descriptor.py

```diff
@@ -53,6 +55,21 @@ def get(self, request):
                             "url": "/extensions/jira/issue/{issue.key}/",
                         },
                     ],
+                    "jiraIssueGlances": [
```

Comment:
@ceorourke do we still need the `webPanels` argument on line 50?

Reply:
@scefali I took it out and it still seems to work 👍